package-lock.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I don't like the idea of introducing npm here. RoR has its own way of getting js.